packages/ui/src/Colors/index.ts

```diff
@@ -1,20 +1,20 @@
 export const Colors = {
-  primary: "#0071f3",
```

Comment:
Looks odd that your linters are set up to fix this in single quote while mine are not.. We might need to check this together with a call at some point, somthing is off

Reply:
That is odd indeed, all mine are single quotes.

Reply:
Lets chat about this situation here sometime, I will clone the repository fresh and install the dependencies and then run the `lint` command. This might give me some insights. I suggest you do the same 👍